packages/dds/migration-shim/src/migrationShim.ts

```diff
@@ -78,6 +87,22 @@ export class MigrationShim extends TypedEventEmitter<IMigrationEvent> implements
 		throw new Error("Method not implemented.");
 	}
 
+	public get target(): LegacySharedTree | ISharedTree {
```

Comment:
Is there a more descriptive name we can use than `target`?  Maybe `currentTree` or `activeTree`?